cmd/scheduled-feed/main.go

```diff
@@ -131,14 +147,16 @@ func main() {
 	}
 }
 
-func cronRequest(port int) {
-	client := &http.Client{
-		Timeout: 10 * time.Second,
-	}
-	resp, err := client.Get(fmt.Sprintf("http://localhost:%v", port))
+func cronRequest(handler *FeedHandler) {
+	pkgs, pollErrors := handler.pollFeeds()
```

Comment:
Does the linter complain if pollErrors is left as `_`? Given it isn't an error type I'm not sure of the linter behaviour

Reply:
The bool returned can be blanked however I've tried to match the behaviour `ServerHTTP` here as to not log/write  `processedPackages` if there's any errors.  We could look to always output the value before returning any errors, if it's not zero?